os/net/mac/framer/frame802154e-ie.h

```diff
@@ -68,6 +68,11 @@ struct tsch_slotframe_and_links {
 /* The information elements that we currently support */
 struct ieee802154_ies {
   /* Header IEs */
+#if MAC_CONF_WITH_CSL
+  uint16_t rendezvous_time;
```

Comment:
I've also been removing padding from structs, and since you have this code fresh in your mind now, is there anything that will break if I put these fields elsewhere in the struct later?

Reply:
The CSL-related variables are safe to reorder.